### PR TITLE
Orders screen

### PR DESCRIPTION
**Pull request by FrederikVil**

### CODE REVIEW

File: lib/screens/category.dart [hunk 1]
```diff
@@ -19,6 +22,72 @@ class CategoryPage extends StatefulWidget {
...
+        orderId = await createOrder(group.id!);
+        order.forEach((element) {
+          createOrderProduct(orderId!, element);
+        });
+        order.clear();
+        Navigator.of(context, rootNavigator: true).pop();
+      },
+    );
+
+    AlertDialog cum = AlertDialog(
```
Comment (kristianbinau) [inline lib/screens/category.dart:63]: Fix placeholder `cum`...

File: lib/services/category.dart [hunk 2]
```diff
@@ -2,6 +2,8 @@ import 'dart:developer';
 
 import 'package:dio/dio.dart';
 import 'package:restaurant_app_flutter/models/category.dart';
+import 'package:restaurant_app_flutter/models/group.dart';
+import 'package:restaurant_app_flutter/models/order.dart';
```
Comment (frederikpyt) [inline lib/services/category.dart:6]:
```suggestion
```

File: lib/screens/category.dart [hunk 3]
```diff
@@ -19,6 +22,72 @@ class CategoryPage extends StatefulWidget {
...
+        ),
+      ),
+    );
+
+    // show the dialog
+    showDialog(
+      context: context,
+      builder: (BuildContext context) {
+        return ListView(
+          children: <Widget>[Flexible(child: alert), Flexible(child: cum)],
```
Comment (kristianbinau) [inline lib/screens/category.dart:86]:
```suggestion
          children: <Widget>[Flexible(child: alert), Flexible(child: alertButtons)],
```

File: lib/screens/category.dart (hunk 1 above)
Comment (kristianbinau) [inline lib/screens/category.dart:63]:
```suggestion
    AlertDialog alertButtons = AlertDialog(
```

**Review by kristianbinau — CHANGES_REQUESTED** [review]

**Review by kristianbinau — APPROVED** [review]
Perfect

**Pull request merged**